README.md

```diff
@@ -258,7 +264,8 @@ IPU ES2K target.
    uses cipher suites as specified in the `inframanager/config.yaml` file. If you
    do not wish to use these default keys, certificates, and cipher suites, then
    modify the `scripts/mev/tls/gen_certs.sh` script accordingly and modify the
-   `inframanager/config.yaml` file with preferred cipher suites.
+   `inframanager/config.yaml` file with preferred cipher suites. This change needs
+   to be done in the previous step to update config.yaml file.
```

Comment:
We should make the "previous step" explicit by giving that step number.
Or, you can move this note to that step (3) itself.

Reply:
done. please check